server/views/pages/sentence.njk

```diff
@@ -124,7 +124,12 @@
                     {% for licenceCondition in sentence.licenceConditions %}
                         {% for lcNote in licenceCondition.notes %}
                             {% set notes %}
-                                <p class="govuk-body">{{ lcNote.note }} </p><p class="govuk-body-s secondary-text">Comment added by {{ lcNote.createdBy }} on {{ lcNote.createdByDate | dateWithYear }}</p>
+                                <p class="govuk-body">{{ lcNote.note }} </p>
+                                {% if lcNote.createdBy %}
+                                    <p class="govuk-body-s secondary-text">Comment added by {{ lcNote.createdBy }} on {{ lcNote.createdByDate | dateWithYear }}</p>
+                                {% else %}
+                                    <p class="govuk-body-s secondary-text">Comment added by not entered</p>
```

Comment:
Would it be better to just hide this text if we don't have the user/date?  Might be worth checking with the designers?

Reply:
will check with them.  i did set to hide the text, but with some notes with the secondary text and some without, i thought it looked a bit wierd